_blog/posts/2017/0.1-announcement.md

```diff
@@ -1,12 +1,17 @@
 ---
-title: "Introducing Istio: A robust service mesh for microservices"
+title: "Introducing Istio"
 overview: Istio 0.1 announcement
-published: true
-permalink: blog/istio-service-mesh-for-microservices.html
-attribution: The Istio Team
-layout: post
+publish_date: May 24, 2017
+subtitle: A robust service mesh for microservices
+attribution:
```

Comment:
Martin Taill.., Louis Ryan, Shriram Rajagopalan. ??

Reply:
Possibly X, Company format? Or should we dump the company?

Reply:
I think "The Istio Team" is more appropriate than 3 individuals for the 0.1 announcement

Reply:
I went with "The Istio Team" for the release announcements.